pkg/credential-helpers/rpm/docker-credential-pass.spec

```diff
@@ -15,7 +15,10 @@ Packager: Docker <support@docker.com>
 Requires: pass
 
 BuildRequires: gcc
+%if ! 0%{?rhel} || (0%{?rhel} && 0%{?_rhn_enabled})
+# libtool-ltdl-devel only available through a Red Hat subscription
 BuildRequires: libtool-ltdl-devel
```

Comment:
Hmm, looks like we might even not need this package anymore.

It was needed by the pkcs11 library, but looks like it's no longer needed after https://github.com/docker/cli/commit/cb3e55bf5802550428867ee8eba0f1a8e3212813

Reply:
Ah that's better then

Reply:
I did a test build and looks like it succeeds.

Also opened a PR to the current docker-ce-packaging to remove the ltdl deps: https://github.com/docker/docker-ce-packaging/pull/1223

Reply:
Opened follow-up https://github.com/docker/packaging/pull/232